ui-admin/src/study/families/FamilyRouter.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import {
+  Link,
+  Route,
+  Routes
+} from 'react-router-dom'
+import { StudyEnvContextT } from '../StudyEnvironmentRouter'
+import { NavBreadcrumb } from 'navbar/AdminNavbar'
+import { StudyEnvParams } from '@juniper/ui-core'
+import FamilyView from './FamilyView'
+import {
+  userHasPermission,
+  useUser
+} from 'user/UserProvider'
+
+/** routes to family views as appropriate */
+export default function FamilyRouter({ studyEnvContext }: {studyEnvContext: StudyEnvContextT}) {
+  const { user } = useUser()
+
+  if (!userHasPermission(user, studyEnvContext.portal.id, 'prototype')) {
```

Comment:
just to confirm -- the thought here is that there won't be a need for a separate "family_view" permission, and so once this feature is finalized, there won't be a permission check here at all?

Reply:
Yes - I am totally open to however we want these perms to work, I assumed something like this wouldn't be locked down since, e.g., viewing participants is not locked down right now